test/controllers/auth/EnforceAmbiguousUserActionSpec.scala

```diff
@@ -55,44 +56,19 @@ class EnforceAmbiguousUserActionSpec extends FreeSpec with MustMatchers with Gui
     "when a user is ambiguous" - {
 
       "return the request it was passed" in {
-        val userRequest =
-          UserRequest(
-            None,
-            None,
-            None,
-            NotEnrolledSelfAssessmentUser(SaUtr("1111111111")),
-            Credentials("", "Verify"),
-            ConfidenceLevel.L50,
-            None,
-            None,
-            None,
-            None,
-            None,
-            None,
-            FakeRequest()
-          )
-        val result = harness()(userRequest)
-        status(result) mustBe OK
+        Seq(NotEnrolledSelfAssessmentUser(SaUtr("1111111111")), WrongCredentialsSelfAssessmentUser(SaUtr("1111111111")))
```

Comment:
I would be tempted to have the map around the test, as it stands if the test fails it won't give you much information about what has failed but if you wrap the test in the map then `.toString` the value in the name of the test and it will help identify which value it fails for